include/deal.II/base/tensor.h

```diff
@@ -886,6 +919,44 @@ constexpr DEAL_II_ALWAYS_INLINE DEAL_II_CUDA_HOST_DEV
 
 
 
+template <int dim, typename Number>
+constexpr DEAL_II_ALWAYS_INLINE DEAL_II_CUDA_HOST_DEV
+                                Tensor<0, dim, Number>::Tensor(const Tensor<0, dim, Number> &other)
+{
+  value = other.value;
```

Comment:
I would prefer if you could move the assignment into the member initializer list. The body of the constructor would be empty then.